packages/ag-charts-community/src/chart/mapping/prepareSeries.ts

```diff
@@ -85,7 +85,7 @@ export function processSeriesOptions(_opts: AgChartOptions, seriesOptions: Serie
 
     const preprocessed = seriesOptions.map((series: SeriesOptions & { stacked?: boolean; grouped?: boolean }) => {
         // Change the default for bar/columns when yKey is used to be grouped rather than stacked.
-        const sType = series.type ?? '';
+        const sType = series.type ?? 'line';
```

Comment:
nit: This should cause a behavioural change (old behaviour: invalid type/error, new behaviour: default to 'line' type). That's probably okay, but it's unclear whether this is what's desired because the change doesn't seemed scoped in bug AG-9212.

Reply:
I've reverted this, I thought it was needed due to tightening of types for the functions called later here (but I've broken that to a distinct PR).

Series without `type` specified are implicitly assumed to be `line` so practically this wouldn't a breaking change.